ols/utils/auth_dependency.py

```diff
@@ -144,6 +155,27 @@ def get_user_info(token: str) -> Optional[kubernetes.client.V1TokenReview]:
         )
 
 
+def get_cluster_id() -> str:
```

Comment:
we already have exactly the same code in your data collector. so perhaps refactor it?

Reply:
Yup, see the `# TODO: use the one from auth_dependency.py` in there.
As I need to most likely update syspath to make it work with import, more testing is involved and I didn't wanted to do it in this PR.